src/wallet/rpcdump.cpp

```diff
@@ -1160,3 +1160,66 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
 
     return response;
 }
+
+UniValue rescanblockchain(const JSONRPCRequest& request)
+{
+    CWallet * const pwallet = GetWalletForJSONRPCRequest(request);
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() > 2) {
+        throw std::runtime_error(
+            "rescanblockchain (\"startheight\") (\"stopheight\")\n"
+            "\nRescan the local blockchain for wallet related transactions.\n"
+            "\nArguments:\n"
+            "1. \"startheight\"    (number, optional) blockheight where the rescan should start\n"
```

Comment:
Just `start`?

Also, if positive then height, if negative then depth relative to tip. Same for `stop`.

Reply:
> if positive then height, if negative then depth relative to tip

In general we don't like overloading arguments with multiple meanings in this way  (I know - I've tried to do it myself before and got NACKed.

The argument names should be changed to `start_height` and `stop_height` to follow style guidelines. I know that this PR was opened before the style changes so I won't suggest that you change variable names, but this is a public interface so it should adhere to the new guidelines.